CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 1.20.3
```

Comment:
Should we have the changelog entries in dev branch first?

Reply:
yeah I think so, this could cause mixup when merging master back to dev

Reply:
I think we need to add change log when merging each PR instead of adding changelog together with release PR tho

Reply:
Did you rebase this branch against dev?

Reply:
Yep. This pr has all changes that dev has now.